### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by jakirkham** [dask/fastparquet#924]
With the recent fix for NumPy 2 in ( https://github.com/dask/fastparquet/pull/922 ), am wondering when the next release might come out

Understand if there are other things that need to be done first. Just trying to get a rough idea

**Comment by martindurant** [comment]
Now on pypi; I suppose a new conda-forge PR will automatically appear and supersede the linked one.

**Comment by jakirkham** [comment]
Thanks Martin! 🙏

So you can either...

1. Request a bot update by filing an issue with this title: [`@conda-forge-admin, please update version`]( https://conda-forge.org/docs/maintainer/infrastructure/#conda-forge-admin-please-update-version )
2. Wait for the bot to make an update PR on its own
3. Make your own update PR to the feedstock

**Comment by martindurant** [comment]
I'll wait :)

**Comment by jakirkham** [comment]
Looks like the bot added a new PR: https://github.com/conda-forge/fastparquet-feedstock/pull/74
